src/seth/default.nix

```diff
@@ -6,10 +6,11 @@ stdenv.mkDerivation rec {
   version = "0.9.0";
   src = ./.;
 
+  buildInputs = [ nodejs ];
   nativeBuildInputs = [makeWrapper shellcheck];
-  buildPhase = "true";
+  dontBuild = true;
```

Comment:
the linking step is not needed? Its being done with `patchShebangs` or?

Reply:
seth doesn't need linking. At runtime, it calls a file named `seth-$subcommand` in libexec, see [here](
https://github.com/dapphub/dapptools/tree/15c241a82183da46f5e170350dd378c99e1f264b/src/seth/libexec/seth/seth#L174)

Reply:
Or what linking step are you referring to?

Reply:
I'm referring to what was hapenning in `make install`

Reply:
The `buildPhase` by default [just runs `make`](https://nixos.org/nixpkgs/manual/#build-phase), which in our case is `test` - so it was running tests, which I found weird.

Reply:
The `buildPhase` above was just running `true` btw, as in the program `true`.